packages/relay/tests/lib/services/transactionService/transactionService.spec.ts

```diff
@@ -208,4 +219,24 @@ describe('Transaction Service', function () {
       );
     });
   });
+
+  describe('calculateTxRecordChargeAmount', () => {
+    it('Should execute calculateTxRecordChargeAmount() to get the charge amount of transaction record', () => {
+      const mockedExchangeRateIncents = 12;
+      const txQueryCostInCents = constants.TX_RECORD_QUERY_COST_IN_CENTS;
+      const hbarToTinybar = Hbar.from(1, HbarUnit.Hbar).toTinybars().toNumber();
+      const expectedTxRecordAmount = Math.round((txQueryCostInCents / mockedExchangeRateIncents) * hbarToTinybar);
```

Comment:
We can use the helper method from `/relay/tests/helpers`

Reply:
Good catch! Thanks!